hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -19,6 +19,15 @@
 from OpenMediaMatch.blueprints import hashing
 
 
+class BankContentResponse(t.TypedDict):
+    id: int
+    disable_until_ts: int
+    collab_metadata: t.Dict[str, t.List[str]]
```

Comment:
nit: you don't have to use the typing version of these
```suggestion
    collab_metadata: dict[str, list[str]]
```